uk_election_timetables/elections/city_of_london_local.py

```diff
@@ -1,14 +1,82 @@
-from datetime import date
+from datetime import date, datetime, timedelta
+from typing import List
 
-from uk_election_timetables.calendars import working_days_before, Country
+from uk_election_timetables.calendars import (
+    working_days_before,
+    Country,
+    UnitedKingdomBankHolidays,
+)
+from uk_election_timetables.date import DateMatcher, WEEKEND
 from uk_election_timetables.election import Election
 
 
-"""
-TODO:
-This is insufficently nuanced
-https://github.com/DemocracyClub/uk-election-timetables/issues/8
-"""
+def _is_bank_holiday(date: date, bank_holidays: List[DateMatcher]) -> bool:
+    for bh in bank_holidays:
+        if bh.matches(date):
+            return True
+    return False
+
+
+def _get_easter_break(bank_holidays: List[DateMatcher]) -> List[DateMatcher]:
+    """
+    “the Easter break” means the period beginning with the Thursday before
+    and ending with the Tuesday after Easter Day
+
+    Fortunately the Gov.UK bank holidays are consistently labelled
+    """
+    easter_break = []
+    for bh in bank_holidays:
+        if bh.name == "Good Friday":
```

Comment:
We need to use something as the "anchor point" to work back/forward from. I could have equally picked "Easter Sunday" here. The choice of "Good Friday" is arbitrary.